go/paddlecloud/config.go

```diff
@@ -40,20 +41,31 @@ func UserHomeDir() string {
 }
 
 func token() (string, error) {
-	// Authenticate to the cloud endpoint
-	authJSON := map[string]string{}
-	authJSON["username"] = config.ActiveConfig.Username
-	authJSON["password"] = config.ActiveConfig.Password
-	authStr, _ := json.Marshal(authJSON)
-	body, err := postCall(authStr, config.ActiveConfig.Endpoint+"/api-token-auth/", "")
+	tokenbytes, err := ioutil.ReadFile(UserHomeDir() + "/.paddle/token_cache")
```

Comment:
On windows, the file path should be `UserHomeDir()\.paddle\token_cache`, so 

```golang
filepath.Join(UserHomeDir(), ".paddle", "token_cache")
```

will adaptor Windows and Unix.